document-generator-core/src/main/java/uk/gov/companieshouse/document/generator/core/DocumentGeneratorApplicationConfiguration.java

```diff
@@ -12,4 +18,19 @@
     EnvironmentReader environmentReader() {
         return new EnvironmentReaderImpl();
     }
+
+    @Bean
+    DocumentInfoService documentInfoService() {
+        return new AccountsDocumentInfoServiceImpl();
+    }
+
+    @Bean
+    TransactionService transactionService() {
+        return new TransactionServiceImpl();
+    }
+
+    @Bean
+    AccountsHandler accountsHandler() {
+        return new AccountsHandlerImpl();
```

Comment:
Probably worth looking into component scanning in Spring in the future to prevent needing the beans defined and might make the autowiring easier to mock in the unit tests.